src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -460,5 +465,23 @@ private static string GetClaimValueType(object obj)
 
             return objType.ToString();
         }
+
+        /// <summary>
+        /// Gets the 'value' corresponding to the provided key from the JWT payload { key, 'value' }.
+        /// </summary>
+        /// <remarks>If the key has no corresponding value, returns null. </remarks>   
+        public T GetPayloadValue<T>(string key)
+        {
+            return Payload.Value<T>(key);
```

Comment:
need null check on 'key'

Reply:
Done. Not sure if it would be better to incline this check with the return statement or not...